documentation/gems.markdown

```diff
@@ -88,6 +88,8 @@ settings with Puppet. You can manage Puppet Server's gem dependencies with the
 package provider shipped in [`puppetlabs-puppetserver_gem`](https://forge.puppet.com/puppetlabs/puppetserver_gem)
 module.
 
+>Note: If you try to load a gem before it's been installed, the agent run will fail with a `LoadError`.
```

Comment:
It seems like the important part of that ticket was the note that the server needs to be restarted after installing the gem. This note might also be useful, but I notice we also don't mention the actual solution to his problem anywhere in this section.